managed/ui/src/routes.js

```diff
@@ -121,6 +121,10 @@ function validateSession(store, replacePath, callback) {
   const userId = Cookies.get('userId') || localStorage.getItem('userId');
   const customerId = Cookies.get('customerId') || localStorage.getItem('customerId');
   if (_.isEmpty(customerId) || _.isEmpty(userId)) {
+    const location = window.location.pathname;
```

Comment:
We should use queryParam to define redirect URL post login. When multiple tabs open local storage doesn't give you consistent behaviour and it creates buggy interaction.

Reply:
Done.